templates/profile/edit.html.twig

```diff
@@ -0,0 +1,26 @@
+{% extends 'layout.html.twig' %}
+
+{% block stylesheets %}
+	{{ parent() }}
+	{{ encore_entry_link_tags('profile') }}
+{% endblock %}
+
+{% block title %}
+	{{ parent() }}
+	- Editer mes informations
+{% endblock %}
+
+{% block body %}
+	{% if app.user and app.user.email == user.email %}
+		<main class="p-5 edit-profile">
+			<h1 class="h1 text-center text-dark">Editer mes informations</h1>
+			<h2 class="h3 text-center text-dark">Ajoutez et mettez à jour vos informations personnelles afin de rendre votre profil visible</h2>
+
+			<div class="d-flex justify-content-center">
+				{{ include('profile/_form.html.twig') }}
+			</div>
+		</main>
+	{% else %}
+		{{ include('error/error403.html.twig') }}
+	{% endif %}
```

Comment:
good practice would be to redirect to a "403" route if someone tries to edit someone else's profile information, and have your "403" page extend your layout so that you can still navigate the site after getting a 403